sdk/test-utils/recorder-new/src/core-v2-recorder.ts

```diff
@@ -41,6 +41,24 @@ export class TestProxyHttpClient {
   public httpClient: HttpClient | undefined = undefined;
   private sessionFile: string | undefined = undefined;
   private sanitizer: Sanitizer | undefined;
+  /**
```

Comment:
```suggestion

  /**
```

The doc comment wants a little bit of space between it and the class fields.